scripts/manage-db.py

```diff
@@ -13,22 +14,77 @@
 
 from auslib.db import AUSDatabase
 
+
+def cleanup_releases(trans, nightly_age, dryrun=True):
+    # This and the subsequent queries use "%%%%%" because we end up going
+    # through two levels of Python string formatting. The first is here,
+    # and the second happens at a low level of SQLAlchemy when the transaction
+    # is being executed.
+    query = """
+LEFT JOIN rules rules_mapping ON (name=rules_mapping.mapping)
+LEFT JOIN rules rules_whitelist ON (name=rules_whitelist.whitelist)
+WHERE name LIKE '%%%%nightly%%%%'
+AND name NOT LIKE '%%%%latest'
```

Comment:
Nit - `%%%%latest%%%%` is used elsewhere but kinda I like `%%%%latest` better. Doubt it makes much real difference but lets pick one to use everywhere.